app/src/main/java/com/spoony/spoony/core/util/extension/ModifierExt.kt

```diff
@@ -228,3 +235,44 @@ fun Modifier.fadingEdge(brush: Brush) = this
         drawContent()
         drawRect(brush = brush, blendMode = BlendMode.DstIn)
     }
+
+data class ShadowSpread(
+    @FloatRange(from = 0.0) val left: Float = 0f,
+    @FloatRange(from = 0.0) val top: Float = 0f,
+    @FloatRange(from = 0.0) val right: Float = 0f,
+    @FloatRange(from = 0.0) val bottom: Float = 0f
+)
+
+fun Modifier.spreadShadow(
+    color: Color = Color.Black,
+    borderRadius: Dp = 0.dp,
+    blurRadius: Dp = 0.dp,
+    spread: ShadowSpread = ShadowSpread()
+): Modifier = this.then(
```

Comment:
p2: composed쓰면 더 좋을것 같아요 Paint객체가 프레임마다 생길거같은데 remember로 줄이는거 어떨지..?

```suggestion
fun Modifier.spreadShadow(
    color: Color = Color.Black,
    borderRadius: Dp = 0.dp,
    blurRadius: Dp = 0.dp,
    spread: ShadowSpread = ShadowSpread()
): Modifier = composed {
    val paint = remember {
        Paint().asFrameworkPaint()
    }
```

Reply:
좋은 의견 감사합니다! 수정했습니다.

Reply:
custom modifier를 구현할 때 [Modifier.Node](https://developer.android.com/develop/ui/compose/custom-modifiers#implement-custom)를 사용해서 만들 수 있대요!!

~~이걸로 바꾸라는거 아님ㅎㅎ 저도 최근에 알게 된거라 잘은 모르는데 일단 공유드립니당~~

Reply:
감사합니다..! 추후에 한번 확인해보겠습니다!